docs/getting-started/quickstart-tt-debugging.md

```diff
@@ -0,0 +1,147 @@
+        "SELECT count(*) FROM dbos_greetings WHERE greeting_name = ?", 
+        [friend]
+    );
+    ctxt.logger.info(`before count ${before.rows[0].count}`);
+
+    await ctxt.client.raw(
+        "INSERT INTO dbos_greetings (greeting_name, greeting_note_content) VALUES (?, ?)",
+        [friend, content]
+    );
+
+    const after = await ctxt.client.raw(
+        "SELECT count(*) FROM dbos_greetings WHERE greeting_name = ?", 
+        [friend]
+    );
+    ctxt.logger.info(`after count ${after.rows[0].count}`);
+}
+```
+
+Now, when we click the `GreetingWorkflow` Code Lens, the workflow execution we select will affect the log output. 
+If we select the oldest execution, we get output that looks like this.
+
+```
+2024-03-22 23:00:53 [info]: Running in debug mode! 
+2024-03-22 23:00:53 [info]: Debugging mode proxy: localhost:2345 
+2024-03-22 23:00:53 [info]: Workflow executor initialized 
+2024-03-22 23:00:56 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:00:56 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:00:56 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:00:56 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:00:56 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:00:57 [info]: before count 0 
+2024-03-22 23:00:57 [info]: after count 1 
+2024-03-22 23:01:01 [info]: Greeting sent to friend! 
+```
+
+But if we select a more recent execution, we get different output.
+
+```
+2024-03-22 23:03:40 [info]: Running in debug mode! 
+2024-03-22 23:03:40 [info]: Debugging mode proxy: localhost:2345 
+2024-03-22 23:03:40 [info]: Workflow executor initialized 
+2024-03-22 23:03:45 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:03:45 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:03:45 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:03:45 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:03:45 [info]: Press control + C to interrupt the workflow... 
+2024-03-22 23:03:46 [info]: before count 2
+2024-03-22 23:03:47 [info]: after count 3 
+2024-03-22 23:03:47 [info]: Greeting sent to friend! 
+```
+
+To clarify what has happened here, you modified the InsertGreeting function to retrieve database state and log it.
+Then, you executed that updated code in the Time Travel Debugger working against _past database state_.
+Note, this worked even though your local code is different from the code running in DBOS Cloud!
+
+:::warning
+When time travel debugging, you can freely add read queries to your application and observe their results when run against past database state.
+This state can be viewed via the logger as described above or via VS Code's variables window.
+However, you cannot change code that updates database state (i.e. insert/delete/update SQL statements) or change the value returned from 
+a workflow or transaction function.
+:::
```

Comment:
At the end, we can link to the debugger cloud tutorial for more complex debugging use cases.